looper/models.py

```diff
@@ -441,6 +441,8 @@ def __init__(self, config_file, subproject=None,
         self.sheet = None
         self.samples = list()
 
+        self.add_sample_sheet()
```

Comment:
Interestingly, it didn't -- sweet! `add_sample_sheet` has idempotence, at least in the context of the test cases where the `Project` fixture that calls this is used.